src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests.cs

```diff
@@ -12,6 +12,7 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public partial class IOperationTests : SemanticModelTestBase
     {
+        [CompilerTrait(CompilerFeature.IOperation)]
```

Comment:
For tuple tests, I opted to put the trait on the class rather than individual tests, where possible. I'd suggest the same here, since the whole file is dedicated to IOperations tests.